runtime/calamari/src/lib.rs

```diff
@@ -928,7 +924,7 @@ mod benches {
         [pallet_parachain_staking, ParachainStaking]
         [pallet_manta_pay, MantaPay]
         // Nimbus pallets
-        [pallet_author_inherent, AuthorInherent]
+        // [pallet_author_inherent, AuthorInherent]
```

Comment:
why disable this pallet?

Reply:
Current the benchmark has issues once open this, need to figure this out.

Reply:
probably just the nimbus branch needs to be updated.